### PR TITLE
feat: add RSS link with icon to page header

### LINKED ISSUE #70: Add RSS icon and link
## Type of Change
- [x] Enhancement
- [ ] Content
- [ ] Improvement / Suggestion
- [ ] Bug
- [ ] Other (please clarify below)

Mostly a reminder to myself or if someone else gets there first: we should really put a link to the RSS feed for the blog posts somewhere (and in the header, if it's not already).

### PR DESCRIPTION
**Pull request by shuuji3**
## Related Issue

resolves #70

## Summary of Changes

Hi, This PR adds an RSS link with the icon to the home header and site header.

The original RSS icon source is from here:  https://commons.wikimedia.org/wiki/File:Generic_Feed-icon.svg

Any feedback is welcome! 🙂 

## Screenshots

(edit: moved to footer, no RSS link in header now)

Screenshot from 2024-03-20 22-53-57
Screenshot from 2024-03-20 22-55-35

### CODE REVIEW

**Comment by rejhgadellaa** [comment]
First of all: Thank you!

Can we maybe move the link to the footer? And maybe have a separate link on the news/blog page if we want it to be more visible there?

**Comment by shuuji3** [comment]
OK, I'll move it to the footer 👍🏻 Do you think it's ok to keep the RSS icon in the footer too?

For the news page, how about placing it on the sidebar like here:
Screenshot from 2024-03-20 23-27-28
https://open-web-advocacy.org/blog/

**Comment by shuuji3** [comment]
Maybe the RSS icon is unnecessary in the footer:

Screenshot from 2024-03-20 23-41-18
Screenshot from 2024-03-20 23-35-53

File: src/_includes/partials/rss-feed.njk [hunk 1]
```diff
@@ -0,0 +1,9 @@
+<div class="rss-feed">
+  <h3>
+    <a href="/feed.xml">
+      <img src="/images/rss.svg">
+      RSS feed
+    </a>
```
Comment (sbesh91) [inline src/_includes/partials/rss-feed.njk:6]: I'd like to see this vertical alignment cleaned up a bit.  In my browser I'd applied these styles to this a tag.
```css
a {
  display: flex;
  gap: 0.25rem;
  align-items: end;
}
```
Alter as needed 👍

Comment (shuuji3) [reply src/_includes/partials/rss-feed.njk:6]: Thanks for the suggestion🙂 

## Before
Screenshot from 2024-03-21 01-08-45

## After
Screenshot from 2024-03-21 01-08-49

This looks much better!

**Review by sbesh91 — APPROVED** [review]
Looks like a great contribution!  Thanks!

File: src/_includes/partials/rss-feed.njk (hunk 1 above)
Comment (sbesh91) [reply src/_includes/partials/rss-feed.njk:6]: Could we write this as `Subscribe to the latest news from OWA`? I think it reads a little better to have `to` in there

**Comment by phae** [comment]
@shuuji3 Thank you!!

**Pull request merged**